mda_openbabel_converter/OpenBabelParser.py

```diff
@@ -4,26 +4,119 @@
 
 import MDAnalysis as mda
 from MDAnalysis.topology.base import TopologyReaderBase
+from MDAnalysis.core.topology import Topology
+import warnings
+
+try:
+    import openbabel as OB
+    from openbabel import OBMol
+    from openbabel import OBElementTable
+except ImportError:
+    print("Cannot find openbabel, install with 'pip install openbabel==2.4.0'")
```

Comment:
```suggestion
    print("Cannot find openbabel, install with 'mamba install -c conda-forge openbabel`)
```
No need to suggest a specific version (unless we know there are some that are incompatible). Generally better to prefer conda / mamba for reasons of binary compatibility with various dependencies (all the packages on conda-forge are built with the same tooling).

Reply:
Got it! That makes sense. Making the change now.

Reply:
Oh true, I didn't consider that one. I'll pop it under the warnings now.